karaf/Dockerfile.rhel

```diff
@@ -27,7 +27,6 @@ USER root
 ADD jboss.repo /etc/yum.repos.d/jboss.repo
 RUN yum install -y --disablerepo=\* --enablerepo=jboss maven \
  && rpm -e --nodeps java-1.7.0-openjdk-devel java-1.7.0-openjdk java-1.7.0-openjdk-headless \
```

Comment:
Would you be able to share the output of `rpm -q maven --requires`? Or whatever the Maven RPM name is.